### PR TITLE
docs: Add examples for awscc_securityhub_finding_aggregator

### PR DESCRIPTION
**Pull request by acwwat**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request
* The resources and data sources in this provider are generated from the CloudFormation schema, so they can only support the actions that the underlying schema supports. For this reason submitted bugs should be limited to defects in the generation and runtime code of the provider. Customizing behavior of the resource, or noting a gap in behavior are not valid bugs and should be submitted as enhancements to AWS via the CloudFormation Open Coverage Roadmap.

### CODE REVIEW

File: templates/resources/securityhub_finding_aggregator.md.tmpl [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+---
+
+# {{.Name}} ({{.Type}})
+
+{{ .Description | trimspace }}
+
+## Example Usage
+
+### Aggregate Findings From All Regions
+
```
Comment (quixoticmonk) [inline templates/resources/securityhub_finding_aggregator.md.tmpl:15]: - Could you include a note in text here that the list of regions cannot contain the calling region in the list ?
- I do believe there is an issue on the finding_aggregator if we switch the `region_linking_mode` from one listing the regions to one without.

Comment (acwwat) [reply templates/resources/securityhub_finding_aggregator.md.tmpl:15]: @quixoticmonk Thanks. I will do some testing this week and update the examples as appropriate.

**Comment by quixoticmonk** [comment]
https://github.com/hashicorp/terraform-provider-awscc/issues/1901 linked to a use case where one switches the region_linking_mode.

**Comment by marcosentino** [comment]
@acwwat Thank you for the contribution!

**Pull request merged**